msteams-platform/sbs-teams-conversation-bot.yml

```diff
@@ -131,15 +137,27 @@ items:
 
          :::image type="content" source="~/assets/images/include-files/output-console-url.png" alt-text="Screenshot shows the url in the Visual Studio output console.":::
 
+         > [!div class="nextstepaction"]
+         > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Create+a+tunnel&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-teams-conversation-bot%3Ftabs%3Ddev%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-teams-conversation-bot.yml%23L66&documentVersionIndependentId=1c9cc6be-8ff3-de89-fa46-6bc355249626&author=surbhigupta&metadata=*%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A*%2BService%253A%2B**msteams**)
```

Comment:
https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/sbs-teams-conversation-bot.yml#L68

Reply:
contentUrl is wrong

Reply:
Line number removed